Package.swift

```diff
@@ -48,6 +50,13 @@ var targets: [PackageDescription.Target] = [
             .product(name: "FilenameMatcher", package: "swift-filename-matcher")
         ]
     ),
+    .target(
+        name: "SourceGraph",
+        dependencies: [
+            .product(name: "SwiftSyntax", package: "swift-syntax"),
```

Comment:
Is this dependency necessary?

Reply:
Right now it's needed only in CommentCommand.swift. Maybe it is a good idea to move somewhere else, what do you think?